index.html

```diff
@@ -128,6 +128,11 @@
 <pre><code>./run-tests.sh</code></pre>
 
 <h2>
+<a name="api-sparkr" class"anchor" href="#api-sparkr"><span class="octicon octicon-link"></span></a>SparkR package API documentation</h2>
+
+<a href="rdocs/00Index.html">Documentation for package ‘SparkR’</a>
```

Comment:
That's the name automatically chosen by knitr. I generally prefer not to mess with things but agree that index is nicer looking.

Re 1.3 - yes that's my thought.

--- Original Message ---

From: "Shivaram Venkataraman" <notifications@github.com>
Sent: March 19, 2015 10:15 PM
To: "amplab-extras/SparkR-pkg" <SparkR-pkg@noreply.github.com>
Cc: "Felix Cheung" <felixcheung_m@hotmail.com>
Subject: Re: [SparkR-pkg] Add SparkR API doc to SparkR github page (#227)

> @@ -128,6 +128,11 @@
>  <pre><code>./run-tests.sh</code></pre>
>
>  <h2>
> +<a name="api-sparkr" class"anchor" href="#api-sparkr"><span class="octicon octicon-link"></span></a>SparkR package API documentation</h2>
> +
> +<a href="rdocs/00Index.html">Documentation for package ‘SparkR’</a>

Instead of `00Index.html` I actually think `index.html` looks better ? Also I guess we should say this is for Spark 1.2 but we can do that when we add docs for Spark 1.3

---
Reply to this email directly or view it on GitHub:
https://github.com/amplab-extras/SparkR-pkg/pull/227/files#r26819199